internal/runner/runner.go

```diff
@@ -349,6 +349,20 @@ func (r *Runner) Test(hr v2.HelmRelease) (*release.Release, error) {
 	test.Namespace = hr.GetReleaseNamespace()
 	test.Timeout = hr.Spec.GetTest().GetTimeout(hr.GetTimeout()).Duration
 
+	filters := make(map[string][]string)
```

Comment:
In the `dev` branch the `runner` will be defunct once I finalize the wiring of the `internal/reconcile` package with the bits in `controllers` (which is the chunk of work I am chewing on at present). Given this, it would be better to move this to `internal/reconcile/test.go` which should be straight forward (and also allows for easier adding of tests :-)).

Reply:
You probably meant to move it to `internal/action/test.go`. That's done now.

Reply:
Oops, that's indeed what I meant.